apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md

```diff
@@ -85,6 +85,21 @@ can in inject the `IManagedTracer` into the action method using the `[FromServic
 
 {{sample:Trace.UseTracerRunIn}}
 
-## Trace Outgoing HTTP Requests
+## Trace Outgoing HTTP Requests (all platforms)
 
 {{sample:Trace.TraceOutgoing}}
+
+## Trace Outgoing HTTP Requests (recommend for Asp.Net Core 2.0 upwards)
+
+The [recommend way of creating HTTPClient](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-2.1) in Asp.Net Core 2.0 and upwards is to use the 
```

Comment:
recommend => recommended (here and in the heading)
HTTPClient => HttpClient
Asp.Net => ASP.NET

Reply:
Done